client/packages/common/src/ui/forms/JsonForms/components/Array.tsx

```diff
@@ -111,6 +113,22 @@ const EnumArrayComponent: FC<EnumArrayControlCustomProps> = ({
         <Box flexBasis={FORM_INPUT_COLUMN_WIDTH}>
           <Autocomplete
             multiple
+            sx={{
+              '.rtl-w7osp1-MuiInputBase-root-MuiInput-root:before': {
```

Comment:
Not sure if possible but is there a different way to style it? Have the feeling this could break as soon as you update the MUI package and the id changes...

Reply:
Yeah, you don't need the id part: https://mui.com/material-ui/customization/how-to-customize/#overriding-styles-with-class-names

In particular:
![Screenshot 2022-12-02 at 5 29 31 PM](https://user-images.githubusercontent.com/5456533/205215097-51c5f10f-7d9c-44c1-a75c-ad0b1a7ecf75.png)

Reply:
[2a43d12](https://github.com/openmsupply/open-msupply/pull/887/commits/2a43d12d1470439d68456c7090fcdcedcb5db606)